gulpfile.js

```diff
@@ -79,14 +85,30 @@ gulp.task("watch-styles", ["default"], () => {
 // Default to clean and build
 
 gulp.task("default", () =>
-    runSequence("clean", "build"));
+    runSequence("clean", "test", "build"));
```

Comment:
I've excluded "lint" from the default build for now since there are 40 errors to clean up. Will do that in a second pass on this PR.